mesonbuild/backend/vs2010backend.py

```diff
@@ -96,6 +97,8 @@ def split_o_flags_args(args):
 def generate_guid_from_path(path, path_type):
     return str(uuid.uuid5(uuid.NAMESPACE_URL, 'meson-vs-' + path_type + ':' + str(path))).upper()
 
+def detectGDK(platform, ignore_case):
+    return re.match(r'Gaming\.(Desktop|Xbox.XboxOne|Xbox.Scarlett)\.x64', platform, re.IGNORECASE if ignore_case else 0)
```

Comment:
nitpicking: we use underscore in method calls: `detect_gdk()`. Also, any reason for not always ignoring case?

Reply:
I had the ignore case toggle because I thought the GDK toolchain in Visual Studio relied on the target name matching a string exactly. Just tested and that is not the case, so I'll remove the parameter.